docker-storage-setup.1

```diff
@@ -36,9 +36,16 @@ The options below should be specified as values acceptable to 'lvextend -L':
 ROOT_SIZE: The size to which the root filesystem should be grown.
 
 DATA_SIZE: The desired size for the docker data LV.  Defaults to using
-           all free space in the VG after the root LV and docker
+           98% free space in the VG after the root LV and docker
            metadata LV have been allocated/grown.
 
+           DATA_SIZE can take values acceptable to "lvextend -L" as
```

Comment:
Just a nit but should we say "lvcreate -L" here since the patch is getting rid of the use of lvextend